src/main/scala/beam/agentsim/agents/rideHail/RideHailingManager.scala

```diff
@@ -90,9 +90,9 @@ class RideHailingManager(
   }
 
   // TODO: remove following (replace by RideHailModifyPassengerScheduleManager)
-  val repositioningPassengerSchedule = mutable.Map[Id[Vehicle], (Id[Interrupt],Option[PassengerSchedule])]()
-  val reservationPassengerSchedule=mutable.Map[Id[Vehicle], (Id[Interrupt],ModifyPassengerSchedule)]()
-  val repositioningVehicles = mutable.Set[Id[Vehicle]]() // TODO: move to RideHailModifyPassengerScheduleManager?
+  val repositioningPassengerSchedule: mutable.Map[Id[Vehicle], (Id[Interrupt], Option[PassengerSchedule])] = mutable.Map[Id[Vehicle], (Id[Interrupt],Option[PassengerSchedule])]()
```

Comment:
Is there actually a bug here? It appears that the code which `put` values is commented out. So all of the contains that WERE here are going to be empty. Same for `reservationPassengerSchedule`